fe/fe-core/src/main/java/com/starrocks/analysis/TableName.java

```diff
@@ -37,6 +37,8 @@
 import java.io.IOException;
 
 public class TableName implements Writable {
+    // as `select` is a key world, which cannot used in SQL, here it denotes a mock table for lambda function.
+    public static final String LAMBDA_FUNC_TABLE = "select";
```

Comment:
Don't get this point.

Reply:
when constructing fields for lambda arguments, a filed requires a table name, which is the `LAMBDA_FUNC_TABLE`. the name shouldnot be the same with user's tables. setting it as `select`  because 'select' is a reserved word, which cannot be user's table name.